website/docs/d/instance_image.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "scaleway"
+page_title: "Scaleway: scaleway_instance_image"
+description: |-
+  Gets information about an Instance Image.
+---
+
+# scaleway_instance_image
+
+Gets information about an instance image.
+
+## Example Usage
+
+```hcl
+// Get info by image name
+data "scaleway_instance_image" "my_image" {
+  name  = "ubuntu bionic"
```

Comment:
Don't feature a use-case like ubuntu. These use-case should be cover by `scaleway_marketplace_image`
Use a name like `my-image-name`